### DISCUSSION TITLE
AttributeError: 'list' object has no attribute 'raw_items'

### DISCUSSION THREAD

**Discussion by l1tecoding** [encode/httpx#2258] [Potential Issue] [answered]
Using the FastAPI I'm deploying the server. 
The class of my main program - accesses another resource to get data, and when trying :

`response = self.client(url=url, headers=headers)`

I get an exception:

`AttributeError: 'list' object has no attribute 'raw_items'`

if I use the **requests** library, everything is fine.

Is there anything I can do to solve this problem?

httpx=0.23.0
hyperframe=6.0.1
h2=4.1.0
uvicorn=0.17.6
httpcore=0.15.0
h11=0.12.0 

I have seen this thread, but have not found a solution to the problem. #1869

**Comment by l1tecoding** [comment, marked as answer]
Example - https://gist.github.com/ilitecoding/d3020d841109f8a99ac1324fd4072c2b

**Reply by lovelydinosaur** [reply]
I think you've got an old version of h11 installed there.
